pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md

```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
 >
 > Please note that once a volume is restored using a snapshot copy, any subsequent files or snapshots will be lost. When a volume is restored, all data in the volume is replaced with the snapshot’s data. The action is irreversible.
 >
-In this guide, a volume is also called “share” as in the OVHcloud API.
 
+In this guide, a volume is also called “share” as in the OVHcloud API.
 
 ## Limitations
 
-It's only possible to restore a volume to its latest available snapshot. However, if you want to restore a volume to an earlier snapshot, you must remove snapshots up to having the snapshot you want to use for restoration be the latest snapshot.
+It's only possible to restore a volume to its latest available snapshot. However, if you want to restore a volume from a previous snapshot, you need to delete snapshots until the snapshot you want to use for the restore is the latest snapshot.
 
-## Scenario 1: Revert volume using manual snapshot
-In this case, you want to restore your volume to its latest manual snapshot, taken through the OVHcloud API or your OVHcloud control panel.
+## Instructions
 
-### Requirements
+### Scenario 1: Reverting a volume using a manual snapshot
 
-- You already did a manual snapshot. If not, you can create a `manual` snapshot through the OVHcloud API or your OVHcloud control panel.
-- The manual snapshot has to belong to the volume you want to restore.
+In this case, you want to restore your volume to its latest manual snapshot, created via the OVHcloud API or the OVHcloud Control Panel.
 
-1\. Identify the latest `manual` volume snapshot using the following API route:
->[!api]
+> [!primary]
+> **Requirements for this scenario:**
+>
+> - You already did a manual snapshot. If not, you can create a `manual` snapshot through the OVHcloud API or your OVHcloud control panel.
+> - The manual snapshot has to belong to the volume you want to restore.
+
+1\. Identify the latest `manual` volume snapshot using the following API call:
+
+> [!api]
 >
 > @api {GET} /storage/netapp/{serviceName}/share/{shareId}/snapshot
 >
+
 - `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore 
 
-
 ![RevertManualSnapshot](images/use_case_1_step_1.png){.thumbnail}
 
-2\. Restore your volume to its latest snapshot by calling `/revert` API route 
+2\. Restore your volume to its latest snapshot using the `/revert` API call: 
 
 > [!api]
 >
 > @api {POST} /storage/netapp/{serviceName}/share/{shareId}/revert
 >
-- `{serviceName}` is the service unique ID 
+
+- `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
-- `{snapshotID}` is the latest share snapshot 
+- `{snapshotID}` is the latest share snapshot
 
-The OVHcloud API will return a HTTP 202 (Accepted) and no response body.
+The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>
```

Comment:
> The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>

 The OVHcloud API will return **an** HTTP 202 (Accepted) code and no response body.<br>